custom/enikshay/management/commands/data_dumps_person_case.py

```diff
@@ -20,6 +20,9 @@ class Command(BaseDataDump):
     https://docs.google.com/spreadsheets/d/1OPp0oFlizDnIyrn7Eiv11vUp8IBmc73hES7qqT-mKKA/edit#gid=1039030624
     """
 
+    TASK_NAME = "data_dumps_person_case"
+    INPUT_FILE_NAME = 'data_dumps_person_case.csv'
```

Comment:
FYI I think Manish just changed this too: https://github.com/dimagi/commcare-hq/pull/19524/files#diff-1b97b1cb3c6b38074e42ad58b8eef419L21